microsetta_public_api/api/taxonomy.py

```diff
@@ -76,3 +76,24 @@ def _present_microbes_taxonomy_table(sample_ids, resource):
     taxonomy_table = taxonomy_.presence_data_table(sample_ids)
     response = jsonify(taxonomy_table.to_dict())
     return response, 200
+
+
+def exists_single(resource, sample_id):
```

Comment:
this and the code below look the same as what was done with alpha, should it be decomposed?

Reply:
I think its fine for now... Its only 7 lines of repeated code, and the decomposition, which we did elsewhere within `validate_resource`, I believe, is much less readable and necessarily couples the interface for `TaxonomyRepo` and `AlphaRepo` which could either be nice, or really annoying later...

Reply:
okay